benefits.md

```diff
@@ -33,13 +33,17 @@ Conferences count as working days (7.5h per day). Transport to conferences count
 
 Try to give notice at least 10-14 days before attending a conference.
 
-Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a bonus for each conference day:
+Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a reimbursal for each away day (conference and travel):
 
 * $50/day for local conferences (in your home town, or close)
-* $100/day for international conferences (you need to fly there, you need a hotel, etc.)
-* $200/day for intercontinental conferences (long flights, extra jetlag days, etc.)
+* $200/day for international conferences (you need to fly there, you need a hotel, etc.)
+* $300/day for intercontinental conferences (long flights, extra jetlag days, etc.)
 * +50%/day if you give a talk on the conference
```

Comment:
Can we clarify the details of this talk, i.e. length, where and when?

Reply:
> length

Talks are from 30min up to an hour. You need to coordinate this with conference organisers.

> where and when

Not sure if I understand this one but I would say "on the conference".

Reply:
I thought it meant a talk about the conference to niteans??

Otherwise it should be 'a talk at the conference'...

Reply:
Yep, it needs to say "Talk at the conference.". You need to be a speaker to get the +50% bonus.